robotcontainer.py

```diff
@@ -165,7 +165,10 @@ def configureButtonBindings(self):
         )
 
         ModifiableJoystickButton(self.operatorInterface.intakeCoral).whileTrue(
-            IntakeCoral(self.intake).repeatedly()
+            commands2.SequentialCommandGroup(
+                ElevatorIntakePosition(self.elevator).repeatedly(),
+                IntakeCoral(self.intake).repeatedly(),
+            )
```

Comment:
1) should be a parallel command group, otherwise the command will repeatedly run the elevator but not the intake
2) this commandgroup should be made into its own command